### PR TITLE
Feat/show all players

### PR DESCRIPTION
**Pull request by anatolegosset**
#31 #34 #33

### CODE REVIEW

File: flaskr/static/show_all_players.js [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+    });
+    table.appendChild(head);
+
+    data.forEach(function(playerObject) {
+        let row = document.createElement('tr');
+        columns.forEach(function(colName) {
+            if (colName != 'Surplus de paiement') {
+                let dataCell = document.createElement('td');
+                if (colName == 'Tableaux') {
+                    let entriesString = "";
```
Comment (layceline) [inline flaskr/static/show_all_players.js:59]: map registeredEntries to categoryId list and join over the latter and add comma between ids

File: flaskr/static/show_all_players.js [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+
+let currentSort = {"colName": null, "ascending": true};
+
+
+function sortByColumn(colName) {
+    if (dontSort.includes(colName)) {
+        return null;
+    }
+    if (colName === null) {
+
```
Comment (layceline) [inline flaskr/static/show_all_players.js:91]: missing something here or delete

File: flaskr/static/show_all_players.js [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+    }
+    currentSort["ascending"] = (colName == currentSort["colName"]) ? (!(currentSort["ascending"])):true;
```
Comment (layceline) [inline flaskr/static/show_all_players.js:93]:
```suggestion
    currentSort["ascending"] = (colName == currentSort["colName"]) ? (!(currentSort["ascending"])) : true;
```

File: flaskr/static/show_all_players.js [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+}
+
+function filterData() {
+    searchString = document.getElementById("players_table_search").value.toLowerCase();
+    filteredArray = [];
+    let cellValue;
+    sortedArray.forEach(function(playerObject) {
+        for (let x in searchCols) {
+            colName = searchCols[x];
+            cellValue = ((numericCols.includes(colName)) ? playerObject[colName].toString():playerObject[colName]).toLowerCase();
```
Comment (layceline) [inline flaskr/static/show_all_players.js:124]:
```suggestion
            cellValue = (numericCols.includes(colName) ? playerObject[colName].toString() : playerObject[colName]).toLowerCase();
```

File: flaskr/static/show_all_players.js [hunk 5]
```diff
@@ -0,0 +1,144 @@
...
+                break;
+            }
+        };
+    })
+    document.getElementById("players_table").remove();
+    putDataInTable(filteredArray, "players_table_div");
+}
+
+
+fetch("http://localhost:5000/api/all_players")
```
Comment (layceline) [inline flaskr/static/show_all_players.js:136]: endpoint should be param

File: flaskr/static/show_all_players.js (hunk 1 above)
Comment (layceline) [reply flaskr/static/show_all_players.js:59]: The category ids must be sorted by start time

File: flaskr/static/styles.css [hunk 6]
```diff
@@ -0,0 +1,29 @@
+h1 {
+color: blue;
+font-size: 60px;
+text-align: center;
+border: 1px solid grey;
+font-family: Tahoma,Verdana,Segoe,sans-serif;
+}
+
+table, th, td {
```
Comment (layceline) [inline flaskr/static/styles.css:9]: the grey lines could be blue `#0366fc`, may need to play with opacity for the text to be readable

**Review by layceline — CHANGES_REQUESTED** [review]

File: flaskr/static/show_all_players.js (hunk 1 above)
Comment (anatolegosset) [reply flaskr/static/show_all_players.js:59]: Sera trié en ordre alphabétique

File: flaskr/static/show_all_players.js (hunk 5 above)
Comment (anatolegosset) [reply flaskr/static/show_all_players.js:136]: rewrote to use relative url if that was the problem?

File: flaskr/static/show_all_players.js [hunk 7]
```diff
@@ -0,0 +1,139 @@
...
+    });
+    table.appendChild(head);
+
+    data.forEach(function(playerObject) {
+        let row = document.createElement('tr');
+        columns.forEach(function(colName) {
+            if (colName != 'Surplus de paiement') {
+                let dataCell = document.createElement('td');
+                if (colName == 'Tableaux') {
+                    console.log(playerObject);
```
Comment (layceline) [inline flaskr/static/show_all_players.js:59]: remove

**Review by layceline — APPROVED** [review]

**Pull request merged**